Runtime/Tracking/Velocity/ComponentTrackerProxy.cs

```diff
@@ -27,20 +26,11 @@ public class ComponentTrackerProxy : VelocityTracker
         /// <inheritdoc />
         public override bool IsActive()
         {
-            return (base.IsActive() && cachedVelocityTracker != null && cachedVelocityTracker.isActiveAndEnabled);
-        }
-
-        /// <summary>
-        /// Sets the source of the <see cref="VelocityTracker"/> proxy.
-        /// </summary>
-        /// <param name="proxySource">The <see cref="GameObject"/> that contains a <see cref="VelocityTracker"/>.</param>
-        public virtual void SetProxySource(GameObject proxySource)
-        {
-            ProxySource = proxySource == null ? null : proxySource.GetComponent<Component>();
+            return base.IsActive() && cachedVelocityTracker != null && cachedVelocityTracker.isActiveAndEnabled;
         }
 
         /// <summary>
-        /// Clears the existing proxy source.
+        /// Clears the existing <see cref="ProxySource"/>.

```

Comment:
We can remove this method now! Malimbe's injected `Clear` method just sets the property to `null` **by using the property setter**, thus the `OnAfterProxySourceChange` below is called and handles this already.